connexion/cli.py

```diff
@@ -42,6 +42,7 @@ def main():
 @click.argument('spec_file')
 @click.argument('base_module_path', required=False)
 @click.option('--port', '-p', default=5000, type=int, help='Port to listen.')
+@click.option('--host', '-H', default=None, type=str, help='Host to listen.')
```

Comment:
You do not need to define `default=False` again here, by default is is already `None`. http://click.pocoo.org/5/api/#click.Parameter

For the help I would add "Host interface to bind on." instead.